dpnp/dpnp_iface_indexing.py

```diff
@@ -46,15 +46,9 @@
 # pylint: disable=no-name-in-module
 from .dpnp_algo import (
```

Comment:
please add tests to `test_sycl_queue.py` and `test_usm_type.py` scopes for each function